.travis.yml

```diff
@@ -19,7 +19,7 @@ before_script:
   - unzip chromedriver_linux64.zip -d /home/travis/bin
 
 install:
-  - pip install selenium
+  - pip install -r requirements-test.txt
```

Comment:
Please just do `pip install selenium request`, requirements file is useful when this is getting complicated, right now, that's not the case.

Also, I forgot that `requests` is not a part of standard library, for something so simple, can please try to use the standard library? I think it's `urllib` (If it not that, ping me, I'll find the default one! :smile: )